packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -915,13 +921,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return root;
     }
 
-    protected serializeContainerBlobs(summaryTreeBuilder: SummaryTreeBuilder) {
+    protected addContainerBlobsToSummary(summaryTree: ISummaryTreeWithStats) {
```

Comment:
>protected [](start = 4, length = 9)

why is this protected?

Reply:
Not sure. This and `getDataStore` are the only protected members. Doesn't look like its for inheritance.
@vladsud added both these methods so he can probably help here.

Reply:
I do not remember adding this method :) Nor I'm sure if the question about them being better private or public?

Reply:
I changed it to private.